.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,46 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        
+      - name: Check_API
+        shell: bash
+        run: |
+          cd scripts
+          ./api_check.sh
+          cd ..  
+        
+      - name: Prerun
```

Comment:
I think we can call this whole section: "Run static analysis"
And this step here: Install static analysis tool
Next " Run static analysis"

Reply:
Separated to a different workflow called `static_analysis`.
Test is called `cppcheck` with `Install`, `Run`, `Upload log` and `Provide correct exit status` steps